src/lib/protocols/rtp.c

```diff
@@ -165,25 +174,83 @@ static u_int8_t isZoom(struct ndpi_flow_struct *flow,
   return(0);
 }
 
+int is_rtp_or_rtcp(struct ndpi_detection_module_struct *ndpi_struct,
```

Comment:
It makes sense to use an `enum` as retval for a better documentation.

Reply:
I agree. Done